com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -579,8 +581,14 @@ public ScriptableRenderer(ScriptableRendererData data)
             Clear(CameraRenderType.Base);
             m_ActiveRenderPassQueue.Clear();
 
+            // TODO: should the asset be passed in to break deps? Maybe we can then rely on the asset, an no need to duplicate state here.
```

Comment:
Feedback? Defensive asset programming here. Replicate data or pass asset forward here? Or just rely that asset always exists. (that will add deps to everywhere).

Reply:
It's not replication as the settings in asset means requested settings, you need to have the runtime platform data settings anyway.